textile/chat-features.textile

```diff
@@ -354,6 +364,59 @@ Broadly speaking, messages are published via REST calls to the Chat HTTP API and
+  ** @(CHA-MR3b3)@ @[Testable]@ For @Distinct@ reaction types, the summary is at @Message.reactions.distinct@ and is of type @Dict<string, SummaryClientIdList>@ (see @TM7b1@).
+  ** @(CHA-MR3b2)@ @[Testable]@ For @Multiple@ reaction types, the summary is at @Message.reactions.multiple@ and is of type @Dict<string, SummaryClientIdCounts>@ (see @TM7b3@).
+
+* @(CHA-MR4)@ @[Testable]@ Sending a reaction is done via the @add@ method of the @MessagesReactions@ object (@room.messages.reactions.add@). The method accepts a message (or message serial), and a @params@ object where the @name@ (ie. emoji string), @type@ (optional) and @count@ (optional, only valid for type @multiple@) are specified.
+** @(CHA-MR4a)@ @[Testable]@ If @type@ is not specified, the default type for the room used.
+
+* @(CHA-MR5)@ @[Testable]@ Users may configure a default message reactions type for a room. This configuration is provided at the @RoomOptions.messages.defaultMessageReactionType@ property, or idiomatic equivalent. The default value is @distinct@.
+
+* @(CHA-MR6)@ @[Testable]@ Subscribing to message reactions is done via the @subscribe@ method of the @MessagesReactions@ object (@room.messages.reactions.subscribe@). The method accepts a callback (or similar idomatic construct) that receives message reactions summary events of type @MessageReactionSummaryEvent@.
+
+* @(CHA-MR7)@ The @MessageReactionSummaryEvent@ interface provides information about reaction summaries for messages.
+  ** @(CHA-MR7a)@ @[Testable]@ The event must include a @type@ field set to indicate it's a reaction summary event.
+  ** @(CHA-MR7b)@ @[Testable]@ The event must include a @summary@ object containing:
+    *** @(CHA-MR7b1)@ @[Testable]@ The @messageSerial@ identifying which message the reactions belong to.
+    *** @(CHA-MR7b2)@ @[Testable]@ A @unique@ field containing reaction summaries for unique-type reactions.
+    *** @(CHA-MR7b3)@ @[Testable]@ A @distinct@ field containing reaction summaries for distinct-type reactions.
+    *** @(CHA-MR7b4)@ @[Testable]@ A @multiple@ field containing reaction summaries for multiple-type reactions.
+
+* @(CHA-MR8)@ @[Testable]@ Subscribing to raw message reactions (as individual annotations) is done via the @subscribeRaw@ method of the @MessagesReactions@ object (@room.messages.reactions.subscribeRaw@). The method accepts a callback (or similar idomatic construct) that receives individual annotations of type @MessageReactionRawEvent@.
+* @(CHA-MR8a)@ @[Testable]@ Subscribing to raw message reactions throws an error if the room is not configured to support raw message reactions by setting room option @RoomOptions.messages.rawMessageReactions@ to @true@ (defaults to @false@).
+
+* @(CHA-MR9)@ The @MessageReactionRawEvent@ interface provides information about individual message reaction events.
```

Comment:
We have talked for a while about having an IDL for Chat. Should we move these points into an IDL section (i.e. an interface definition of the type), and perhaps just have spec points cover specific behvaiour we expect for each one (beyond plain old data)?

Reply:
Moved those next to where the example data for them is given as well.